src/pg.py

```diff
@@ -748,7 +748,7 @@ def replication_slots(self, action, slots):
                 if not self._create_replication_slot(slot):
                     return False
             else:
-                if current and slot not in current:
+                if current is not None and slot not in current:
```

Comment:
`current` is never `None`, it can only be an empty list: https://github.com/yandex/pgconsul/pull/30/files#diff-1dd21d0846b6c985aaf231799772c4797a69429020c8d3781caf8e285d35db43R148

Reply:
https://github.com/yandex/pgconsul/blob/71f48d1fb44716daa1e9e0aa76d136f55abc0bb7/src/pg.py#L145

current is None if we catch an error
Actually, i do not remember specific error, but i saw it in feature tests before i write this line